### PR TITLE
Sergi e/p098 ros

### PR DESCRIPTION
**Pull request by sergi-e**
#### Description

- Style iteration for the whole "ROS" section.
- Updates on quick start installation regarding the new packages for CARLA 0.9.8 and CARLA 0.9.7 (with "all main"). 

#### Where has this been tested?

  * **Platform(s):** Ubuntu 18.04.4
  * **Python version(s):** 3.6.9 and 2.7.17
  * **Unreal Engine version(s):** 4.22

#### Possible Drawbacks

- Links inside tables may not work online. Check after merge.
- New launchfiles regarding ScenarioRunner to be added.

---
This change is [Reviewable](https://reviewable.io/reviews/carla-simulator/carla/2591)

### CODE REVIEW

File: Docs/ros_launchs.md [hunk 1]
```diff
@@ -3,17 +3,15 @@
...
 ## carla_ackermann_control.launch
 
-Creates a node to manage a vehicle using Ackermann controls instead of the CARLA control messages. The node reads the vehicle info from CARLA and uses it to define the controller. A simple Python PID is used to adjust acceleration/velocity. It can be installed using: 
+Creates a node to manage a vehicle using Ackermann controls instead of the CARLA control messages. The node reads the vehicle info from CARLA and uses it to define the controller. A simple Python PID is used to adjust acceleration/velocity. 
 ```sh
 pip install --user simple-pid
 ```
-Initial parameters can be set in: `share/carla_ackermann_control/config/settings.yaml`.  
-It is possible to modify the parameters during runtime via ROS dynamic reconfigure.  
+Initial parameters can be set in `share/carla_ackermann_control/config/settings.yaml`. It is possible to modify the parameters in runtime via ROS dynamic reconfigure.
```
Comment (himanshugoswami) [inline Docs/ros_launchs.md]: At "Initial parameters can be set in `share/carla_ackermann_control/config/settings.yaml`."
The path should be something like

If you installed ROS-Bridge using APT repository '/opt/carla-ros-bridge/melodic/share/carla_ackermann_control/config/settings.yaml'

or If you installed using source repository '<carla-ros-bridge PATH>/catkin_ws/src/ros-bridge/carla_ackermann_control/config/settings.yaml'

File: Docs/ros_launchs.md [hunk 2]
```diff
@@ -46,7 +45,7 @@ Spawns an ego vehicle with sensors attached and waits for world information.
 
 ---
 ## carla_example_ego_vehicle.launch
-Based on [carla_ego_vehicle.launch](#carla-ego-vehicle-launch), spawns an ego vehicle (`role-name="ego_vehicle"`) using a provided file describing the sensors. Said file can be found in: `share/carla_ego_vehicle/config/sensors.json`. 
+Based on [carla_ego_vehicle.launch](#carla-ego-vehicle-launch), spawns an ego vehicle (`role-name="ego_vehicle"`) using a provided file describing the sensors. Said file can be found in `/share/carla_ego_vehicle/config/sensors.json`.
```
Comment (himanshugoswami) [inline Docs/ros_launchs.md]: At "Said file can be found in `/share/carla_ego_vehicle/config/sensors.json`."
The path should be something like

If you installed ROS-Bridge using APT repository '/opt/carla-ros-bridge/melodic/share/carla_ego_vehicle/config/sensors.json'

or If you installed using source repository '<carla-ros-bridge PATH>/catkin_ws/src/ros-bridge/carla_ego_vehicle/config/sensors.json'

Comment (sergi-e) [reply Docs/ros_launchs.md]: Updated!

File: Docs/ros_launchs.md (hunk 1 above)
Comment (sergi-e) [reply Docs/ros_launchs.md]: Updated!

**Pull request merged**